src/app/components/fireside/stream/video/video.vue

```diff
@@ -1,11 +1,25 @@
 <script lang="ts" src="./video"></script>
 
 <template>
-	<div class="-video-player" />
+	<div class="-video-player">
+		<div ref="video" :class="{ '-hide': !shouldPlayVideo }" />
+		<canvas ref="canvas" :class="{ '-hide': shouldPlayVideo }" />
```

Comment:
Why can't we do `v-when` instead of this? I guess you don't wanna `v-if` to not unload it.

Reply:
I changed it to use `v-show` on the canvas, and removed it on the video. The video hides itself already.